docs: CNCF onboarding (#236)
bank-vaults/bank-vaults.dev: content/docs/support.md

## Community support

> If you encounter problems while using Bank-Vaults that the documentation does not address, [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component, or talk to us in the [#bank-vaults channel of the CNCF Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).

Reviewer: I'd insert "you can" in front of [open an issue].

The comma is not necessary before "or talk to us..."

> If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component or talk to us in the [#bank-vaults channel of the CNCF Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).

Reviewer 2: ```suggestion
If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component or talk to us in the [#bank-vaults channel of the Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).
```

> If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component or talk to us in the [#bank-vaults channel of the CNCF Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).

Reviewer 2: ```suggestion
If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue]([https://github.com/bank-vaults/bank-vaults/issues](https://github.com/bank-vaults/bank-vaults/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen)) or write us on [Slack]({{< relref "/docs/support.md" >}}).
```

> If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component or talk to us in the [#bank-vaults channel of the CNCF Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).

Reviewer 2: ```suggestion
If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component or write us on [#bank-vaults channel on Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).
```
